04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,63 @@
+const Product = require('../models/product')
+
+
+const getAllProductsStatic = async (req,res) => {
+    const search = 'ab'
+    const products = await Product.find({price:{$gt: 30}}).sort('price').select('name price')
+    res.status(200).json({ products, ngHits: products.length })
```

Comment:
question (my own curiousity): why are we using the key `ngHits` to indicate the size of the results?